src/dmd/backend/cod3.d

```diff
@@ -2438,29 +2446,29 @@ else
 
 void load_localgot(ref CodeBuilder cdb)
 {
-static if (TARGET_LINUX || TARGET_FREEBSD || TARGET_OPENBSD || TARGET_DRAGONFLYBSD || TARGET_SOLARIS)
-{
-    if (config.flags3 & CFG3pic && I32)
+    static if (TARGET_LINUX || TARGET_FREEBSD || TARGET_OPENBSD || TARGET_DRAGONFLYBSD || TARGET_SOLARIS)
     {
-        if (localgot && !(localgot.Sflags & SFLdead))
-        {
-            localgot.Sflags &= ~GTregcand;     // because this hack doesn't work with reg allocator
-            elem *e = el_var(localgot);
-            regm_t retregs = mBX;
-            codelem(cdb,e,&retregs,false);
-            el_free(e);
-        }
-        else
+        if (config.flags3 & CFG3pic && I32)
         {
-            elem *e = el_long(TYnptr, 0);
-            e.Eoper = OPgot;
-            regm_t retregs = mBX;
-            codelem(cdb,e,&retregs,false);
-            el_free(e);
+            if (localgot && !(localgot.Sflags & SFLdead))
+            {
+                localgot.Sflags &= ~GTregcand;     // because this hack doesn't work with reg allocator
+                elem *e = el_var(localgot);
+                regm_t retregs = mBX;
+                codelem(cdb,e,&retregs,false);
+                el_free(e);
+            }
+            else
+            {
+                elem *e = el_long(TYnptr, 0);
+                e.Eoper = OPgot;
+                regm_t retregs = mBX;
+                codelem(cdb,e,&retregs,false);
+                el_free(e);
+            }
         }
     }
 }
-}
```

Comment:
Not whitespace either, but it's hard to see which opening brace it's matching ?

Reply:
That's just github diff getting confused with the change in indentation.